README.md

```diff
@@ -136,3 +136,28 @@ If you do not have Python 3.10 installed, you can use root-less podman with a Py
 ```bash
 podman run --rm --tty --volume "$(pwd):/mnt:z" --workdir /mnt docker.io/library/python:3.10 bash -c 'pip install nox ; nox -s pip-compile'
 ```
+
+## Creating release tags
+Creating tags that correspond with releases and release candidates in [`ansible/ansible`](https://github.com/ansible/ansible)
+
+First, install the additional tagging dependencies
+
+```bash
+pip install gitpython typer
+```
+
+Next, you'll need to ensure that you have both [`ansible/ansible`](https://github.com/ansible/ansible) and [`ansible/ansible-documentation`](https://github.com/ansible/ansible-documentation) checked out in sibling directories.
+
+Lastly, run the various tasks with the tagging tool.
+
+Show tags from ansible-core that are not present here:
+```bash
+./hacking/tagger/tag.py new-tags
+```
+
+Determine missing `anisble-core` releases from `--branch`. Create corresponding tags for releases in `ansible-documentation`, and push them:
```

Comment:
TBH I'm not sure what you mean with the first sentence.

Reply:
Yeah, I don't know how to clearly express this. I have the code bases checked out at ~/projects/ansible/ansible and ~/projects/ansible/ansible-documentation (so, they're sibling dirs, at the same level). But `~/projects` is just my convention, and I added `ansible/` in there to keep all related projects together. 

It *seems* like the initial request for this implied that both code bases needed to be checked out like that, but the tool seems to find them. I'll play with it a bit and see how forgiving it is and then work on cleaning this up.

Reply:
After some testing on the taggers auto detection of repository locations, I've updated this section and added a bit more to the "run" bit showing how they'd be used.

Reply:
I was referring to this sentence: "Determine missing `anisble-core` releases from `--branch`."

Could you please explain this a bit more? I don't think the tagger has a `--branch` parameter so I'm not sure what that refers to.